bfdriver/src/platform/efi/.gitignore

```diff
@@ -0,0 +1,5 @@
+src/*.o
```

Comment:
Can we move these the top-level .gitignore with the rest of the driver exclusions?